src/infrastructure/model/resolutionEntryDTO.ts

```diff
@@ -42,7 +42,7 @@ export class ResolutionEntryDTO {
         {
             "name": "resolved",
             "baseName": "resolved",
-            "type": "AnyOfstringstring"
+            "type": "any"
```

Comment:
instead of patching the generated code, I'm patching the open api spec. The generated code in java is untouched, not even committed. Maybe is better?

Reply:
Will probably refactor it after RC